pulsar-client/src/main/java/org/apache/pulsar/client/api/ReaderBuilder.java

```diff
@@ -137,4 +137,11 @@
      * @param readerName
      */
     ReaderBuilder readerName(String readerName);
+
+    /**
+     * Set the subscription role prefix.
+     *
+     * @param subscriptionRolePrefix
+     */
+    ReaderBuilder subscriptionRolePrefix(String subscriptionRolePrefix);
```

Comment:
The Javadoc should mention that the default prefix is `reader`

Reply:
Added explanation:)